README.md

```diff
@@ -108,6 +108,16 @@ Recase.to_title("some-value") # => "Some Value"
 Recase.to_title("some value") # => "Some Value"
 ```
 
+### Enumerable
+
+Enumerable applies convert on keys of a map
+
+```elixir
+Recase.Enumerable.convert_keys(%{"some-value" => "value"}, &Recase.to_pascal/1) # => %{"SomeValue" => "value"}
```

Comment:
Let's use better examples here. The most common use-case for it will be `camelCase` <-> `snake_case` conversion (well, that what I see the most in Elixir)

It is better to directly give this example here.

Reply:
I left the 3 examples? it is right ?